nncf/experimental/torch/fx/transformations.py

```diff
@@ -593,7 +837,7 @@ def separate_linear_and_bias(model: torch.fx.GraphModule):
     for n in model.graph.nodes:
         if not _is_linear(n):
             continue
-        if len(n.args) < 3 or n.args[2] is None:
+        if len(n.args) < 3 or n.args[2] is None or n.args[1].op != "get_attr":
```

Comment:
What's the purpose of this check?

Reply:
https://github.com/openvinotoolkit/nncf/pull/2984#discussion_r1802694964